docs/package.json

```
@@ -6,6 +6,7 @@
     "dev": "next dev -p 5000",
     "build": "next build && next export",
     "lint": "next lint",
+    "postbuild": "next-sitemap",
```

The build is failing likely because this isn't being executed with the correct `$PATH`:

```suggestion
    "postbuild": "yarn next-sitemap",
```

---

oof, added now. thanks!